ATTRIBUTIONS/wasabi-attribution.xml

```diff
@@ -611,4 +611,4 @@
          </downloadUrls>
       </dependency>
    </dependencies>
-</attributionReport>
```

Comment:
This file changed automatically by build... What change do you suggest to do here?